packages/provider/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  equatable:
 
```

Comment:
Why add this dependency?

Reply:
Mostly for unit testing states. For example, I can trigger and action, then assert that the states that were emitted match my expectations. This is 100% option, just wanted to through it in here for discussion

```dart
      subject.logIn('token');
      //wait for whatever
      expect(states, [
        AuthenticationState(isLoading: true, isAuthenticated: false),
        AuthenticationState(isLoading: false, isAuthenticated: true),
      ]);
```